[2025-04-15T20:23:33.181-0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-15T20:23:33.185-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: multi_city_weather_pipeline.fetch_weather_for_all_cities scheduled__2025-04-14T23:00:00+00:00 [queued]>
[2025-04-15T20:23:33.187-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: multi_city_weather_pipeline.fetch_weather_for_all_cities scheduled__2025-04-14T23:00:00+00:00 [queued]>
[2025-04-15T20:23:33.187-0500] {taskinstance.py:2867} INFO - Starting attempt 3 of 2
[2025-04-15T20:23:33.191-0500] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_weather_for_all_cities> on 2025-04-14 23:00:00+00:00
[2025-04-15T20:23:33.194-0500] {standard_task_runner.py:72} INFO - Started process 9328 to run task
[2025-04-15T20:23:33.199-0500] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'multi_city_weather_pipeline', 'fetch_weather_for_all_cities', 'scheduled__2025-04-14T23:00:00+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/weather_pipeline_dag.py', '--cfg-path', '/var/folders/4n/q_76d_d57mb9q1qtrf4mvh940000gn/T/tmpqvbo6esb']
[2025-04-15T20:23:33.200-0500] {standard_task_runner.py:105} INFO - Job 83: Subtask fetch_weather_for_all_cities
[2025-04-15T20:23:33.216-0500] {task_command.py:467} INFO - Running <TaskInstance: multi_city_weather_pipeline.fetch_weather_for_all_cities scheduled__2025-04-14T23:00:00+00:00 [running]> on host ramyas-macbook-air.local
[2025-04-15T20:23:33.236-0500] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='multi_city_weather_pipeline' AIRFLOW_CTX_TASK_ID='fetch_weather_for_all_cities' AIRFLOW_CTX_EXECUTION_DATE='2025-04-14T23:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-14T23:00:00+00:00'
[2025-04-15T20:23:33.236-0500] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-15T20:23:33.236-0500] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-15T20:23:33.237-0500] {logging_mixin.py:190} INFO - Current task name:fetch_weather_for_all_cities state:running start_date:2025-04-16 01:23:33.185929+00:00
[2025-04-15T20:23:33.237-0500] {logging_mixin.py:190} INFO - Dag name:multi_city_weather_pipeline and current dag run status:running
[2025-04-15T20:23:33.237-0500] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-15T20:27:48.934-0500] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-04-15T20:27:48.940-0500] {process_utils.py:132} INFO - Sending 15 to group 9328. PIDs of all processes in the group: [9328]
[2025-04-15T20:27:48.941-0500] {process_utils.py:87} INFO - Sending the signal 15 to group 9328
[2025-04-15T20:27:53.149-0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-15T20:27:53.151-0500] {taskinstance.py:2604} INFO - Dependencies not met for <TaskInstance: multi_city_weather_pipeline.fetch_weather_for_all_cities scheduled__2025-04-14T23:00:00+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2025-04-15T20:27:53.154-0500] {taskinstance.py:2604} INFO - Dependencies not met for <TaskInstance: multi_city_weather_pipeline.fetch_weather_for_all_cities scheduled__2025-04-14T23:00:00+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2025-04-15T20:27:53.155-0500] {local_task_job_runner.py:166} INFO - Task is not able to be run
